helper/debug.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const _ = require('lodash');
+
+class Debug {
+  constructor(moduleName){
+    this.name = moduleName || 'unnamed module';
+  }
```

Comment:
I see, I guess if there are lots of calls in the same file that a developer could simply:

```javascript
const debug = new Debug( 'foo' );
const push = debug.push.bind( debug, req );
```